docs/source/usersguide/input.rst

```diff
@@ -558,19 +559,127 @@ The following code snippet illustrates the instantiation of the ``CPUSolver`` fo
+
+.. _figure_fixed_source_flux_calc:
+     
+.. table:: 
+   
+   +--------------------------------------------------------+--------------------------------------------------------+
+   | .. _figa:                                              | .. _figb:                                              |
+   |                                                        |                                                        |
+   | .. image:: ../../img/fs-flux-calc-4-angles-group-1.png | .. image:: ../../img/fs-flux-calc-32-angles-group-1.png|
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+While this case seems illsuited for the ``computeFlux(...)`` routine, the ``computeFlux(...)`` would be very useful for the fixed source cosine distribution problem described previously above. This problem
+
+TODO: make fixed source plots
+
+Source Calculations
+-------------------
+
+In other problems, the source distribution is desired for a set eigenvalue. For this case, the ``computeSource(...)`` routine can be used, which calculates the total source (including any fixed source) in each flat source region iteratively. At the end of each transport sweep, the eigenvalue is reset to the eigenvalue set by the user. By default this value is 1.0 if no value is provided by the user. Returning to the first problem discussed using the ``computeFlux(...)`` routine where a fixed source was placed in a geometry filled with water, the ``computeFlux(...)`` routine could not account for scattering. Since the ``computeSource(...)`` routine computes both scattering and fission sources during transport sweeps, it is able to account for scattering and compute the correct flux distribution. The line below shows how the ``computeSource(...)`` routine can be called.
+
+.. code-block:: python
+
+  solver.computeSource(max_iters)
+
+The resulting flux distribution in the third energy group (which previously was calculated to be zero everywhere) is shown below using 4 azimuthal angles in the figure to the left and 32 azimuthal angles in the figure to the right. Notice that ray effects are still present when a low number of azimuthal angles are used, but the effects are far less extreme than observed with ``computeFlux(...)`` due to scattering and with 32 azimuthal angles, the ray effects seem to have disappeared.
```

Comment:
I think the last sentence could be made stronger by rewording: "the ray effects seem to have disappeared" -> "the ray effects have largely disappeared"